nuxt.config.js

```diff
@@ -25,7 +25,7 @@ export default defineNuxtConfig({
 
   // currently we can only use nitro in development https://github.com/nuxt/framework/issues/886
   bridge: {
-    nitro: process.env.NODE_ENV !== 'production',
+    nitro: false,
```

Comment:
What's this?

Reply:
whoops, played around with new nuxt-edge version. just reintroduced it